src/types/journey.ts

```diff
@@ -8,17 +8,19 @@ export enum CrowdingLevel {
     High = 3,
 }
 
+export type NavigationKind = "depart-at" | "arrive-by" | "depart-after";
```

Comment:
It would be nice to define this type only once and import it wherever else it's needed. I think this might be the best file to put it in.